event/src/unvalidated/payload/init.rs

```diff
@@ -5,7 +5,6 @@ use serde::{Deserialize, Serialize};
 #[derive(Serialize, Deserialize)]
 pub struct Payload<D> {
     header: Header,
-    #[serde(skip_serializing_if = "Option::is_none")]
```

Comment:
unit test failed without this change.  Looks like unsigned init events sent by js-ceramic have an explicit `data: null` in them.

Reply:
ha, so you did unit test this! could add a few test structs to build a carfile that fails, but it seems like we are testing the matches/success case already